lib/enkf/analysis_config.cpp

```diff
@@ -44,7 +44,7 @@
 #include <ert/enkf/model_config.hpp>
 
 
-#define UPDATE_OVERLAP_KEY      "OVERLAP_LIMIT"
+#define UPDATE_ENKF_ALPHA_KEY      "ENKF_ALPAHA"
```

Comment:
May I suggest that we change it to simply `ALPHA`, instead of `ENKF_ALPAHA`

Reply:
hmmm... i'm not sure... ? if people have used enkf_alpha in their configs up until now - i guess we will brake it by changing name? 

feel free to hijack and fix if possible - always nice to get rid of `enkf` if it makes sense !

Reply:
I agree. Let us keep it as is 👍